Dockerfile

```diff
@@ -10,10 +10,10 @@ FROM python:3.8-alpine
 
 WORKDIR /app
 
-COPY Pipfile* docker-entrypoint.sh ./

```

Comment:
Why separate the copy? Each command in the dockerfile creates another layer, so this is just adding extra redundant layers.

Reply:
I mostly separated the COPY and RUN commands for development purposes so I didn't have to sit through Docker installing all the packages if/when I changed the docker-entrypoint.sh file. I'm not opposed to merging them back if that's your preference.